code/__DEFINES/jobs.dm

```diff
@@ -228,6 +228,22 @@ DEFINE_BITFIELD(departments_bitflags, list(
 /// This job is a head of staff.
 #define JOB_HEAD_OF_STAFF (1<<12)
 
+DEFINE_BITFIELD(job_flags, list(
+	"JOB_ANNOUNCE_ARRIVAL" = JOB_ANNOUNCE_ARRIVAL,
```

Comment:
do you think this should be alphabetized (since the actual order of bitflags are irrelevant due to the fact that they're all defines anyways?)

Reply:
I do it like this cause it's easier for me to tell if one is missing but I can change it if you want

Reply:
probably okay then